### DISCUSSION TITLE
Modern Standby and AniMe Matrix doesn't go along?

### DISCUSSION THREAD

**Discussion by Scarlaid** [seerge/g-helper#3369] [General]
Initially, when I first got the G14 with Matrix, I was still using Armoury Crate and the "sleep" matrix thing still worked whenever I closed the lid, despite having S0 Standby State

Now it just doesn't work anymore, so I did some testing
- With ROG Logo option, the matrix remains active and does its animation upon closing the lid
- With a custom GIF (very light, around 2-3MB), the animation will stop upon closing the lid

Before uninstalling Armoury Crate, I did have this enabled, hence why it worked
![image](https://github.com/user-attachments/assets/c0d380c8-c66c-4f67-acba-79a0597b27d6)

I have also added all three configurations
![image](https://github.com/user-attachments/assets/abc0ae43-05d7-48a7-a8c5-12c7fc684ed9)

Nothing prevents my laptop from going to "sleep"
![Screenshot 2024-11-12 183257](https://github.com/user-attachments/assets/1ec95086-8bc0-4ada-b9c6-e6b187068a51)

I also have not tried any tweaks relating to the S0/S3 sleep state, I am too scared that might brick Windows
What am I doing wrong here? Is there a way to enable this power-saving matrix animation again?

**Comment by seerge** [comment]
@Scarlaid hello,

Do you have any “disable on lid close” option turned on? Cause then it will shut down matrix completely 

Otherwise - sleep animation is controlled by firmware. As soon as device enters sleep from firmware/hardware point of view - it would enable sleep animation for matrix and blinking animation for keyboard (if it’s checked under Extra). So to double-check if device is sleeping- check keyboard animation. And also power led will be blinking

**Reply by Scarlaid** [reply]
so I reinstalled Armoury Crate and re-check all the Matrix options, they are all double-checked at On, not Off, and both the "Lid Close" and "Sleep" checkboxes are enabled

do the testing and to my sadness, sleeping doesn't show the animation, only the ROG Logo was shown, the rest are fine

I do realize that, as you mentioned about the blinking keyboard, it doesn't blink, nor does the first power light, unlike the first time I used the laptop the first week when the keyboard and the light still blink, the back showed the Starlight animation

I do not see any "disable on lid close" option too, only "Turn off animation when unplugged", that is the same as "Disable on battery"

**Reply by Scarlaid** [reply]
I'm now on G-Helper and it still doesn't work

**Reply by seerge** [reply]
@Scarlaid because if power led is not blinking - device is not sleeping from hardware point of view:) windows may be acting. Check sleepstudy if it shows any reasons

**Reply by Scarlaid** [reply]
So what I did was restarting the laptop so it would not have anything opened, only Powertoy, Defender, IDM and G-Helper are on startup
Then I ran a sleep test for 45 mins until it goes into hibernation
![image](https://github.com/user-attachments/assets/9f05f374-2a04-44c9-b83b-2a19f9c971d7)

I'm just weirded out, what is this "audio bus" thing though 🤔 I legit opened nothing
There are no blinking animation nor sleeping animation

**Reply by Scarlaid** [reply]
So it was very simple

Device Manager > Find the Audio Bus > Disable it

my laptop back to sleep now haha, it was indeed something wrong with the latest Windows update that made this running consistently 

Audio still working well so idk what that does, just happy that this was resolved

**Reply by Scarlaid** [reply]
Thanks for telling me about the SleepStudy :D

**Reply by seerge** [reply]
@Scarlaid i’m glad you have figured it out :)

**Reply by IceStormNG** [reply]
Btw: The same issue also happens with AC when using the "Audio Effect". Reason is that it hooks the Audio device and therefore keeps the device handle awake. There is a command for Windows to ignore a device and go to sleep anyways. I used that back then to fix this so it will go to sleep anyways with low power draw... sadly, I forgot the command 😓
